playbooks/roles/ocp-metallb-operator/tasks/install.yml

```diff
@@ -0,0 +1,293 @@
+---
+# tasks file for playbooks/roles/ocp-metallb-operator
+
+- name: Check if cluster operators and nodes are healthy
+  include_role:
+    name: check-cluster-health
+
+- name: Clean up
```

Comment:
Can you have a separate file to perform clean up the operator to improve reusability?

Reply:
Moved to a separate file